tests/end_to_end/README.md

```diff
@@ -0,0 +1,161 @@
+# wazuh-qa
+
+Wazuh - Quality assurance automation templates
```

Comment:
copy, paste? I think it is not appropriate. Instead, you could put an introduction to what E2E tests are and, perhaps, the classification we make within this structure.